build/__pythonVersions.sh

```diff
@@ -1,7 +1,7 @@
 # This file was auto-generated from 'constants.yaml'. Changes may be overridden.
 
 PYTHON_BASE_TAG='20200217.4'
-PIP_VERSION='20.0.2'
+PIP_VERSION='20.1'
```

Comment:
As you know pip is also built from source just like python and included within it but currently our PlatformBinaries pipeline does not build an sdk version if it is already present in the dev storage account. This was an optimization to avoid building sdks if they are really required.
There is a environment variable called `OVERWRITE_EXISTING_SDKS` which you can set when queuing a build in this pipeline and as the name indicates it will cause all sdks to be built again, but since it is going to trigger ALL platforms' sdk builds, I would suggest to introduce platform specific variable (like `OVERWRITE_EXISTING_PYTHON_SDKS`) to override or we could even create separate pipelines for each platform (yeah this is lot of work)

Reply:
Introducing the environment variable like `OVERWRITE_EXISTING_PYTHON_SDKS` sounds plausible for this case, so it will shorten the overall build time and we want to create one for each platform by editing the pipeline script.. And one thing I noticed maybe unrelated to the above is that we have `Oryx-baseImages-PlatformName ` builds for each platform except Python platform, should we create one for Python?..

Reply:
No we do not need base image build for Python.